Source/KSCrash/Source/KSCrash/Recording/Sentry/BSG_KSCrashSentry_CPPException.mm

```diff
@@ -174,9 +222,8 @@ static void CPPExceptionTerminate(void) {
             @"Crash handling complete. Restoring original handlers.");
         bsg_kscrashsentry_uninstall((BSG_KSCrashType)BSG_KSCrashTypeAll);
         bsg_kscrashsentry_resumeThreads();
+        bsg_g_originalTerminateHandler();
```

Comment:
According to [this documentation](http://www.cplusplus.com/reference/exception/set_terminate/) the returned pointer from `std::setTerminate(fn)` can be a `null` pointer.